### DISCUSSION TITLE
[Question] What are the default directory/file permissions on Linux

### DISCUSSION THREAD

**Discussion by mwasson74** [grafana/grafana#71292] [General]
Someone has updated the directory and file permissions on our Linux server and I'd like to put them back to the default. Could someone please let me know what they are by default for any and all locations of Grafana-related directory and files? 🙏

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
